### PR TITLE
WebGLTextures: Used sized internal format for depth stencil

### LINKED ISSUE #16756: gl.DEPTH_STENCIL not accepted by renderbufferStorageMultisample()
The value DEPTH_STENCIL here is rejected as invalid in Chrome 75 (but accepted by Firefox).

https://github.com/mrdoob/three.js/blob/6737d0dfe3a92a83f0a7e385e223bae437bce091/src/renderers/webgl/WebGLTextures.js#L795

..looking into the specs for the underlying openGL call at https://www.khronos.org/registry/OpenGL-Refpages/es3.0/html/glRenderbufferStorageMultisample.xhtml

a sized internalFormat such as DEPTH24_STENCIL8 is required, Firefox translates DEPTH_STENCIL to DEPTH24_STENCIL8 internally.

Is changing this to DEPTH24_STENCIL8 acceptable?

### PR DESCRIPTION
**Pull request by aardgoose**
Fixed #16756

gl.renderbufferStorageMultisample() requires a sized internal format specifier.  Chrome enforces this, firefox extends the behaviour of renderbufferStorage():

`To be backward compatible with WebGL 1, also accepts internal format DEPTH_STENCIL, which should be mapped to DEPTH24_STENCIL8 by implementations.`

to renderbufferStorageMultisample() which is not required by the spec.

### CODE REVIEW

**Comment by Mugen87** [comment]
Before merging this we might want to wait for some feedback here: https://github.com/mrdoob/three.js/issues/16756#issuecomment-501213428

**Comment by aardgoose** [comment]
Ok, from reading the spec Chrome is compliant, Firefox has just extended the laxity of WebGL1 to WebGL2 functions (internally to FF both renderbufferStorage variants take exactly the same path where the translation of DEPTH_STENCIL to DEPTH24_STENCIL8 takes place, before hitting ANGLE etc ).

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by Mugen87** [comment]
@aardgoose Since you introduced a new constant, it's necessary to add it right here:

https://github.com/mrdoob/three.js/blob/dev/rollup.config.js#L3

Otherwise you can a warning when performing the build:

> * Unhandled GL Constant: DEPTH24_STENCIL8

**Comment by mrdoob** [comment]
Fixed https://github.com/mrdoob/three.js/commit/ab130804c713ae87280856ab6653a4ebbd7181d1

**Comment by aardgoose** [comment]
@Mugen87 
Oops! Apologies, I should have spotted that, I was aware of the constant post processing.

**Pull request merged**
